src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -40,12 +41,18 @@ export const FirstSuggestionItem: React.FC<SuggestionItemProps> = ({
   return (
     <SuggestionItemLink
       bg={isHighlighted ? "black5" : "white100"}
-      borderBottom="1px solid"
-      borderBottomColor="black10"
+      mt={1}
+      borderTop="1px solid"
+      borderTopColor="black10"
       onClick={handleClick}
       to={href}
     >
-      <Text variant="sm">See full results for &ldquo;{query}&rdquo;</Text>
+      <Flex alignItems="center">
+        <Text variant="sm" mr={1}>
+          See full results for &ldquo;{query}&rdquo;
+        </Text>
+        <ChevronButton />
```

Comment:
**suggestion**: I think it's better to use `ChevronRightIcon` here. Something like this

```javascript
import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
...
<ChevronRightIcon height="14px" />
```